diff --git a/importlib_metadata-4.8.1/setup.cfg b/importlib_metadata-4.8.1/setup.cfg
index ac885fda219..0fc22e28984 100644
--- a/importlib_metadata-4.8.1/setup.cfg
+++ b/importlib_metadata-4.8.1/setup.cfg
@@ -1,5 +1,6 @@
 [metadata]
 name = importlib_metadata
 author = Jason R. Coombs
+version = 4.8.1
 author_email = jaraco@jaraco.com
 description = Read metadata from Python packages
 long_description = file:README.rst